test/e2e/serial/tests/configuration.go

```diff
@@ -693,9 +697,222 @@ var _ = Describe("[serial][disruptive] numaresources configuration management",
 				return kcCmNamesCur
 			}).WithContext(ctx).WithTimeout(5 * time.Minute).WithPolling(10 * time.Second).Should(Equal(kcCmNamesPre))
 		})
+
+		It("[reboot_required][slow][unsched][schedrst] should be able to correctly identify topology manager policy without scheduler restarting", Label("reboot_required", "slow", "unsched", "schedrst"), func(ctx context.Context) {
+			// https://issues.redhat.com/browse/OCPBUGS-34583
+			fxt.IsRebootTest = true
+			By("getting the number of cpus that is required for a numa zone to create a topology affinity error deployment")
+			const (
+				NUMAZonesRequired     = 2
+				hostsRequired         = 1
+				cpuResourcePercentage = 1.5 // 1.5 meaning 150 percent
+			)
+			var nrtCandidates []nrtv1alpha2.NodeResourceTopology
+			By(fmt.Sprintf("filtering available nodes with at least %d NUMA zones", NUMAZonesRequired))
+			nrtCandidates = e2enrt.FilterZoneCountEqual(nrtList.Items, NUMAZonesRequired)
+			if len(nrtCandidates) < hostsRequired {
+				e2efixture.Skipf(fxt, "not enough nodes with 2 NUMA Zones: found %d", len(nrtCandidates))
+			}
+
+			// choosing the first node because the nodes should be similar in topology so it doesnt matter which one we choose
+			referenceNode := nrtCandidates[0]
+			referenceZone := referenceNode.Zones[0]
+
+			cpuQty, ok := e2enrt.FindResourceAvailableByName(referenceZone.Resources, string(corev1.ResourceCPU))
+			Expect(ok).To(BeTrue(), "no CPU resource in zone %q node %q", referenceZone.Name, referenceNode.Name)
+
+			cpuNum, ok := cpuQty.AsInt64()
+			Expect(ok).To(BeTrue(), "invalid CPU resource in zone %q node %q: %v", referenceZone.Name, referenceNode.Name, cpuQty)
+			klog.Infof("available CPUs per numa on the node: %d, ", cpuNum)
```

Comment:
nit: let's use https://kubernetes.io/blog/2020/09/04/kubernetes-1-19-introducing-structured-logs/ in new code (Infof -> InfoS)

Reply:
np will do!